src/Ilios/CliBundle/Command/UpdateFrontendCommand.php

```diff
@@ -159,23 +180,30 @@ public function isOptional()
     }
 
     /**
-     * @param string $cacheDir
      * @param string $environment
-     * @param string $version
+     * @param string|bool $versionOverride
      *
      * @throws \Exception
      */
-    protected function writeIndexFile($cacheDir, $environment, $version)
+    protected function downloadAndExtractArchive($environment = 'prod', $versionOverride = false)
     {
-        if (!$this->keepFrontendUpdated) {
-            $version = $this->releaseVersion;
+        $fileName = $this->apiVersion . '/' . self::ARCHIVE_FILE_NAME;
+        if ($versionOverride) {
+            $fileName .= ':' . $versionOverride;
         }
-
-        $contents = $this->builder->getIndex($environment, $version);
-        if (!$contents) {
-            throw new \Exception('Unable to build the index file');
+        $url = self::PRODUCTION_CDN_ASSET_DOMAIN;
+        if ($environment === self::STAGING) {
+            $url = self::STAGING_CDN_ASSET_DOMAIN;
         }
-
-        $this->fs->dumpFile($cacheDir . '/' . self::CACHE_FILE_NAME, $contents);
+        $string = $this->fetch->get($url . $fileName);
+        $archiveDir = $this->temporaryFileStorePath;
+        $archivePath = $archiveDir . self::ARCHIVE_FILE_NAME;
```

Comment:
needs '/' between archive dir and archive file name.

Reply:
Fixed, thanks.